### ISSUE TITLE
IOS RadListView ng-template with custom component (+ ListViewGridLayout ) crash the app

### ISSUE THREAD

**Issue by PraktikerNeosoft** [NativeScript/nativescript-ui-feedback#1042]
I have IOS problem in RadListView. The list view works perfectly as long as I have inline view component declared like below
```
<GridLayout [visibility]="!isLoading && !isError ? 'visible' : 'collapse'"
                      class="item-boxes" height="100%">
    <RadListView [items]="categoryItems" pullToRefresh="true"
                            (pullToRefreshInitiated)="onPullToRefreshInitiated($event)"
                             loadOnDemandMode="Auto
                            (loadMoreDataRequested)="onLoadMoreItemsRequested($event)">
        <ng-template tkListItemTemplate let-item="item">
            <FlexboxLayout class="item-box">
                 <Label [text]="item.name"></Label>
            </FlexboxLayout>
        </ng-template>
        <ListViewGridLayout tkListViewLayout
                                            ios:itemHeight="200"
                                            scrollDirection="Vertical"
                                            spanCount="2"></ListViewGridLayout>
    </RadListView>
</GridLayout>
```
I'd like to make each item-box separate view so I create ItemBoxComponent. Android works fine this code but ios crash app. No error out in console. What can I do? Thanx for your help.

```
<GridLayout [visibility]="!isLoading && !isError ? 'visible' : 'collapse'"
                      class="item-boxes" height="100%">
    <RadListView [items]="categoryItems" pullToRefresh="true"
                            (pullToRefreshInitiated)="onPullToRefreshInitiated($event)"
                             loadOnDemandMode="Auto
                            (loadMoreDataRequested)="onLoadMoreItemsRequested($event)">
        <ng-template tkListItemTemplate let-item="item">
            <ItemBox [item]="item"></ItemBox>
        </ng-template>
        <ListViewGridLayout tkListViewLayout
                                            ios:itemHeight="200"
                                            scrollDirection="Vertical"
                                            spanCount="2"></ListViewGridLayout>
    </RadListView>
</GridLayout>
```
Miklos

**Comment by NickIliev** [comment]
@PraktikerNeosoft thank you for the details information!  I was able to reproduce the issue by using your snippets [in this demo app](https://github.com/NickIliev/NS-Issues-2019-I/tree/master/nativescript-ui-feedback/issue-1042) so marking this one as a bug.

Steps to reproduce:
- clone and run [the app ](https://github.com/NickIliev/NS-Issues-2019-I/tree/master/nativescript-ui-feedback/issue-1042)no iOS
- The app immediately crashes if we use custom component which is not wrapped in another layout or if we use `ListViewGridLayout ` - happening only on iOS - Android works as expected.

**Comment by PraktikerNeosoft** [comment]
@NickIliev Lot of thnx. My list layout has 2 columns so i'd like to use ListViewGridLayout. 
I'm waiting for bugfix. Miklos

**Comment by PraktikerNeosoft** [comment]
@NickIliev We're waiting for the bugfix yet.
Could you tell me when is expected to come?
It would be very important to finalize our application.

I look forward to receiving your reply.
Balint

**Comment by tgpetrov** [comment]
Released with [nativescript-ui-listview@7.0.1](https://github.com/NativeScript/nativescript-ui-feedback/blob/master/releases/listview.md#701-2019-july-17)
